src/components/Resizer/Resizer.spec.jsx

```diff
@@ -1,17 +1,21 @@
-jest.mock('./Resizer.util');
+jest.mock('element-resize-detector', () => {
+	return function() {
+		return {
+			listenTo: jest.fn((_element, handleResize) => {
+				handleResize({
+					offsetWidth: 50,
+					offsetHeight: 100,
+				});
+			}),
+			removeListener: jest.fn(),
+		};
+	};
+});
 import _ from 'lodash';
 import React from 'react';
 import { common } from '../../util/generic-tests';
 import { mount } from 'enzyme';
 import Resizer from './Resizer';
-import { elementResizeDetector } from './Resizer.util';
```

Comment:
We only pulled this out to a `util` to maintain the singleton while making testing a little easier.

If we're okay with it not being a singleton, we don't need it anymore.